tf_encrypted/keras/__init__.py

```diff
@@ -1,12 +1,13 @@
 """Higher-level layer abstractions built on TF Encrypted."""
 from __future__ import absolute_import
 
-from . import engine
-from . import layers
-
+from tf_encrypted.keras import engine
+from tf_encrypted.keras import layers
+from tf_encrypted.keras.engine.sequential import Sequential
```

Comment:
`Sequential` should be in `tfe.keras.models` instead of `tfe.keras.engine` per both TF and Keras

Reply:
this is not the case: https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/python/keras/engine/sequential.py
it just happens to be publicly available from that location

Reply:
public API location is what matters, not the file it lives in